tests/test_pyclipper.py

```diff
@@ -392,6 +392,16 @@ def test_paths_scale_from(self):
         assert all(isinstance(k, float) for i in res for j in i for k in j)
 
 
+class TestNonStandardNumbers(TestCase):
+    
+    def test_sympyzero(self):
+        from sympy import Point2D
```

Comment:
This test will fail when `sympy` won't be available. I suggest skipping this test when this import is not available (some info on how to do it https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures).

Eg
```
        try:
            import sympy
            # test here
        except ImportError:
            self.skipTest("Skipping, sympy not available")
```